Britannia/localization/english/BTWK_buildings_l_english.yml

```diff
@@ -141,3 +157,31 @@
  building_type_antonine_wall_01_desc:0 "The Antonine Wall was constructed around a quarter of a century after Hadrian's Wall, during the reign of the Emperor Antoninus Pius. While it was intended to be a new, more northernly Caledonian frontier, it was occupied for only a few years before its desertion. "
  building_antonine_wall_01:0 "$building_type_antonine_wall_01$"
  building_antonine_wall_01_desc:0 "$building_type_antonine_wall_01_desc$"
+
+ ################
+ # BATH'S BATHS #
+ ################
+ building_type_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_type_btwk_aquae_sulis_baths_01_desc:0 "During the Roman conquest of Britain, the Romans built a series of baths around some natural springs from which the local Britons worshipped their God #italic Sulis #!(hence, the name of the settlment #italic Aquae Sulis#!). Over the following centuries during Roman occupation, this settlement would grow into a large resort complex, of which its baths being a staple feature."
+
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths"
+ building_btwk_aquae_sulis_baths_01_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nThe departure of Romans from Britain shifted of the focus of local polities to prioritising the security of their realms, rather than to the upkeep of amenities like this spa. As such, the once magnificent resort has fallen into disuse and decay, and stands more as a monument to an age long past."
+ building_btwk_aquae_sulis_baths_01:0 "Roman Baths - Restored"
+ building_btwk_aquae_sulis_baths_02_desc:0 "$building_type_btwk_aquae_sulis_baths_01_desc$ \n\nRenovations have been carried out on the spa resort to allow for the recreational bathing the Romans once enjoyed. This has encouraged a larger influx of freemen to bring their custom to the resort - a great boon to the local economy."
+
+
+ ###################
+ # GLASTONBURY TOR #
+ ###################
+ building_type_btwk_glastonbury_tor_01:0 "Glastonbury Tor"
+ building_type_btwk_glastonbury_tor_01_desc:0 "The pre-Christian history of the tor is quite unknown. Archeological evidence indicates there was no permanent settlement until after around the 6th century AD, and would later give way to Glastonbury Abbey. Arthurian legend, however, suggests this as being the homestead of the mythical druid Merlin..."
```

Comment:
Talking about "pre-Christian history" and "Archeological evidence" definitely sounds a bit weird here
"Arthurian legend" and "mythical druid" is also kind of weird wording considering that Merlin literally owns the county at the start
I would probably just talk about this being an important religious site, and then something about how there are many legends surrounding the Tor, including stuff about Joseph Arimathea/the Holy Thorn and the Isle of Avalon.

Reply:
Wasn't sure what to write about it to be honest. I'm not well versed in the myths surrounding it. Wrote it in the frame of reference of today looking back in history. I can have a look at those.